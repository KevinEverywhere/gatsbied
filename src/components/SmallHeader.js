import React from 'react'
import SectionChosen from './SectionChosen'

const SmallHeader = (props) => (
    <header id="header" style={props.timeout ? {display: 'none'} : {}}>
      <SectionChosen { ...props } />
    </header>
)

export default SmallHeader
