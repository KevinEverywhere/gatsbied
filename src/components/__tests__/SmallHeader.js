import React from "react"
import renderer from "react-test-renderer"
import SmallHeader from "../SmallHeader"

describe("SmallHeader", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<SmallHeader />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
