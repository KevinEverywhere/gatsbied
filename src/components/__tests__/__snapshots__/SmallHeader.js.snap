// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallHeader renders correctly 1`] = `
<header
  id="header"
  style={Object {}}
>
  <nav>
    <a
      href="/"
    >
      <div
        className="smlogo"
      >
        <span
          className="icon fa-bicycle"
        />
      </div>
    </a>
    <ul>
      <li>
        <a
          href="work"
        >
          Work
        </a>
      </li>
      <li>
        <a
          href="creative"
        >
          Creative
        </a>
      </li>
      <li>
        <a
          href="contact"
        >
          Contact
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
